src/view/com/posts/Feed.tsx

```diff
@@ -115,15 +136,20 @@ export const Feed = observer(function Feed({
 
   const renderItem = React.useCallback(
     ({item}: {item: any}) => {
-      if (item === EMPTY_FEED_ITEM) {
-        return renderEmptyState()
-      } else if (item === ERROR_ITEM) {
+      if (item === FEED_INNER_HEADER_LOADING) {
+        return <FeedHeaderLoading />
+      } else if (item === FEED_INNER_HEADER) {
         return (
-          <ErrorMessage
-            message={feed.error}
-            onPressTryAgain={onPressTryAgain}
+          <FeedInnerHeader
+            showContextMenu={showFeedHeaderContextMenu}
```

Comment:
This prop is for use on the `CustomFeed` screen, which already uses this context menu.